CONTRIBUTING.md

```diff
@@ -1,74 +1,78 @@
 I will assume familiarity with the Go toolchain.
 
 ## Getting Set Up
+
 1. **Star the repository**: Since you're about to contribute to the project, why not give us a star to show your support? �
 2. **Install the latest [Go](https://go.dev/dl/)**: We generally attempt to stay up-to-date with the latest version of Go.
-For the specific version in use, you may check the [go.mod](https://github.com/dolthub/doltgresql/blob/main/go.mod) file.
+   For the specific version in use, you may check the [go.mod](https://github.com/dolthub/doltgresql/blob/main/go.mod) file.
 3. **Install PSQL 15**: Although we do not use PostgreSQL, we do use the PSQL client (for [PostgreSQL 15](https://www.postgresql.org/download/)) for testing.
-In addition, it is worthwhile to have a local PostgreSQL instance installed to cross-check behavior, especially when writing tests (which should _always_ be verified by a [PostgreSQL 15](https://www.postgresql.org/download/) instance).
+   In addition, it is worthwhile to have a local PostgreSQL instance installed to cross-check behavior, especially when writing tests (which should _always_ be verified by a [PostgreSQL 15](https://www.postgresql.org/download/) instance).
 4. **Clone the repository**: Clone the repository to a local directory of your choice.
 5. **Build the parser**: Run the `doltgresql/postgres/parser/build.sh` script.
-This creates a few files within the `doltgresql/postgres/parser/parser` directory that are necessary for parsing PostgreSQL statements.
-It is recommended to run this file every time you pull changes into your local repository, as these generated files are not included since they would cause near guaranteed merge conflicts.
+   This creates a few files within the `doltgresql/postgres/parser/parser` directory that are necessary for parsing PostgreSQL statements.
+   It is recommended to run this file every time you pull changes into your local repository, as these generated files are not included since they would cause near guaranteed merge conflicts.
 6. **Run Go tests**: Before building the project, you should always run all of the tests, which can be done by running `go test ./... --count=1` from the source root directory.
-This ensures that all Go tests pass, which also ensures that your Go environment is installed and configured correctly.
-7. **Build the binary**: From the source root directory, run `go build -o <bin_name> .`, where `<bin_name>` is the name of the binary (usually `doltgres` or `doltgres.exe`).
-   To run the program without creating an executable, run `go run .`.
+   This ensures that all Go tests pass, which also ensures that your Go environment is installed and configured correctly.
+7. **Build the binary**: From the source root directory, run `go build -o <bin_name> ./cmd/doltgres`, where `<bin_name>` is the name of the binary (usually `doltgres` or `doltgres.exe`).
+   To run the program without creating an executable, run `go run ./cmd/doltgres`.
```

Comment:
Oops my formatter got this but I only changed this part

Reply:
All good, the document is in serious need of love anyway. I should spend some time next week polishing it up.